06_modules_lab/02.py

```diff
@@ -1,4 +1,16 @@
-""" Write a program that reads 2 numbers from sys.argv
-and prints their sum.
-"""
-
+"""
+The program receives two numbers as command line arguments
+And prints their sum.
+
+If the user did not enter exactly 2 command line arguments,
+The usage is printed to the screen. 
+"""
+import sys
+
+if (len(sys.argv) != 3):
+    print "Usage: {} <firstNum> <secondNum>", sys.argv[0]
+    sys.exit()
+
+(prName, firstNum, secondNum) = sys.argv

```

Comment:
Python has a special variables for data nobody cares about: `_`. A cleaner line is therefore:
`(_, firstNum, secondNum) = sys.argv`